mindjunkies/forums/tests/test_views.py

```
@@ -1,197 +1,186 @@
 import pytest
-from django.test import TestCase, RequestFactory, Client
-from django.urls import reverse
 from django.contrib.auth import get_user_model
 from django.contrib.messages.storage.fallback import FallbackStorage
 from django.http import HttpRequest
+from django.test import Client, RequestFactory, TestCase
```

Ruff (0.8.2)

`django.test.Client` imported but unused

Remove unused import: `django.test.Client`

(F401)